includes/Integrations/AMP.php

```diff
@@ -232,7 +232,7 @@ public function add_amp_content_sanitizers( $sanitizers ) {
 	 * @param array     $error Validation error being sanitized.
 	 * @return null|bool Whether sanitized.
```

Comment:
Should this be flipped  ?

bool|null

Reply:
Thisis unrelated to my change and should be done in another PR

Reply:
I think we shouldn't change this. Seems fine as it is.